chart/kubeapps/templates/kubeops-deployment.yaml

```diff
@@ -48,6 +48,12 @@ spec:
           args:
             - --user-agent-comment=kubeapps/{{ .Chart.AppVersion }}
             - --assetsvc-url=http://{{ template "kubeapps.assetsvc.fullname" . }}:{{ .Values.assetsvc.service.port }}
+            {{- if .Values.featureFlags.additionalClusters }}
+            - --additional-clusters-config-path=/config/additional-clusters.conf
+          volumeMounts:
```

Comment:
I would split the `volumeMounts` in a different `if` in case we need to mount something else in the future.

Reply:
Yeah, I wondered about that, but thought we can do that when needed. Anyway, updated.